src/components/MoreInformation.js

```diff
@@ -53,12 +55,100 @@ function ExtraData({ data }) {
-                    <b>{key}:</b> {data[key]}
-                </div>
-            );
+            //formats additional information instead of just adding it to the more information section
+            if (key === "Additional Information") {
+                const currentData = data[key];
+                if (currentData) {
+                    var finalData = parseMD(currentData);
+                    //adds the parsed data as straight html
+                    elements.push(
+                        <div
+                            className={classes.extraData}
+                            dangerouslySetInnerHTML={{
+                                __html: `<b>${key}:</b> ${finalData}`,
+                            }}
+                        />
+                    );
+                }
+            } else {
+                elements.push(
+                    <div className={classes.extraData}>
+                        <b>{key}:</b> {data[key]}
+                    </div>
+                );
+            }
         }
         return elements;
     }
 }
+
+function parseMD(currentData) {
+    //encodes angle brackets to escape data
+    let workingData1;
+    let workingData2;
+    let indexOfLAB = currentData.indexOf("<");
+    let indexOfRAB = currentData.indexOf(">");
+    if (indexOfLAB !== -1) {
+        workingData1 = currentData.replaceAll("<", "&#60;");
+    } else {
+        workingData1 = currentData;
+    }
+    if (indexOfRAB !== -1) {
+        workingData2 = workingData1.replaceAll(">", "&#62;");
+    } else {
+        workingData2 = workingData1;
+    }
+    //parses bold italics first, splitting data at each "***"
+    let splitDataEmStr = workingData2.split("***");
+    let newDataEmStr = splitDataEmStr[0];
+    let isEmStr = false;
+    //iterates through the split data
+    //uses em to italicize and span with the secondary color for bolding (went with a color change since the titles were bold)
+    let i = 0;
+    for (i = 1; i < splitDataEmStr.length; i++) {
+        if (isEmStr) {
+            newDataEmStr = newDataEmStr + "</span></em>" + splitDataEmStr[i];
+            isEmStr = false;
+        } else {
+            newDataEmStr =
+                newDataEmStr +
+                '<em><span style="color: #F50057">' +
+                splitDataEmStr[i];
+            isEmStr = true;
+        }
+    }
+    //parses bold next, splitting data at each "**"
+    let splitDataStr = newDataEmStr.split("**");
+    let newDataStr = splitDataStr[0];
+    let isStr = false;
+    //iterates through the split data
+    //same formatting rules as 85
+    for (i = 1; i < splitDataStr.length; i++) {
+        if (isStr) {
+            newDataStr = newDataStr + "</span>" + splitDataStr[i];
+            isStr = false;
+        } else {
+            newDataStr =
+                newDataStr +
+                '<span class="formattedExtraData">' +
+                splitDataStr[i];
+            isStr = true;
+        }
```

Comment:
Same thing, if you want to:

```suggestion
        if (isStr) {
            newDataStr = newDataStr + splitDataStr[i];
            isStr = false;
        } else {
            newDataStr = newDataStr + `<span class="formattedExtraData">${splitDataStr[i]}</span>`;
            isStr = true;
        }
```

Also, do you know why we're using a span with a class for bold instead of the `<strong></strong>` HTML element?